lib/phoenix/code_reloader.ex

```diff
@@ -49,26 +51,86 @@ defmodule Phoenix.CodeReloader do
   API used by Plug to invoke the code reloader on every request.
   """
   def call(conn, opts) do
-    case opts[:reloader].(conn.private.phoenix_endpoint) do
-      :ok ->
+    {conn, res, output} = call_reloader(conn, opts[:reloader])
+
+    case {res, feedback_started?(conn)} do
+      {:ok, true} ->
+        Plug.Conn.chunk(conn, "Done!<br />\n")
+        Plug.Conn.chunk(conn, ~s(<meta http-equiv="refresh" content="0">\n))
```

Comment:
I’ve added an `is_unix?` check to the `send_feedback?` test, so this feature won’t be active on Windows.